exonum/src/storage/values.rs

```diff
@@ -282,6 +283,25 @@ impl StorageValue for DateTime<Utc> {
     }
 }
 
+/// Uses little-endian encoding.
+impl StorageValue for Duration {
```

Comment:
You need to check whether it actually possible to use `Duration` inside `encoding_struct` - I think we also need `ExonumJson` here?